packages/store/internals/src/initial-state.ts

```diff
@@ -0,0 +1,17 @@
+import { InjectionToken } from '@angular/core';
+
+export const INITIAL_STATE_TOKEN = new InjectionToken<any>('INITIAL_STATE_TOKEN');
+
+export class InitialState {
```

Comment:
`InitialState` to `StaticInitialState` // analogue StaticInjector from Angular

Reply:
I don't think it adds readability